.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           sudo cp fitsverify /usr/local/bin
       - name: Install Python dependencies
         run: |
-          pip install roman-datamodels==0.1.2
+          pip install roman-datamodels==0.5.0
```

Comment:
Should we stop pinning this now?  That way we'll start seeing test failures when roman-datamodels changes again.

Reply:
i will once the test pass and I am sure I can deploy to test, right now its pinned to whats on dev and what will be on test.